pkgs/applications/office/csv2odf/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, python3, fetchurl }:
+
+let
+  inherit (python3.pkgs) buildPythonApplication;
+in
+
+buildPythonApplication rec {
```

and just get rid of the let bindings?

```suggestion
python3.pkgs.buildPythonApplication rec {
```